.env

```diff
@@ -87,7 +87,7 @@ _APP_LOGGING_PROVIDER=
 _APP_LOGGING_CONFIG=
 _APP_GRAPHQL_MAX_BATCH_SIZE=10
 _APP_GRAPHQL_MAX_COMPLEXITY=250
-_APP_GRAPHQL_MAX_DEPTH=3
+_APP_GRAPHQL_MAX_DEPTH=4
```

Comment:
Why this change? The updated tests should only be 3 levels 🤔

Reply:
There's some counting error when calculating depth in GraphQL;